### ISSUE TITLE
Have the up/down buttons in the calculation use common code

### ISSUE THREAD

**Issue by jonathanolson** [phetsims/area-model-common#152]
Should be better for a11y and behavior. We may want to mix in "tweaker" behavior also?

**Comment by jonathanolson** [comment]
Notes I had internally:
```
Use ArrowButton in CalculationNode (instead of those custom arrow nodes) -- better for a11y
	Use NodesPushButton
OR use a ButtonModel, and see RectagularPushButton for how it would be integrated with a11y.
```

Not sure if we should use NodesPushButton, do something more manual, or look into more general common components.

**Comment by amanda-phet** [comment]
I had some brief email/slack exchanges with @terracoda about this so before you change it, let's talk about some of her ideas for designing for a11y in case it impacts your changes. Changing the button type might not make a huge difference for a11y because one thought is to use accessibleSlider. If we make it a "slider", users can use the up/down arrow keys to navigate up and down, rather than tabbing from the up arrow to the down arrow.

**Comment by jonathanolson** [comment]
I'll hold off. I'll definitely be capable of making it act like a slider for accessibility if desired.
